### PR TITLE
Model

### PR DESCRIPTION
**Pull request by tpazderka**

### CODE REVIEW

File: django_oidc_user/models.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models
+
+
+class OidcUserSub(models.Model):
+    """Store user and sub."""
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, related_name='oidc_sub', on_delete=models.CASCADE)
```
Comment (ziima) [inline django_oidc_user/models.py]: Related name should be in plural - `oidc_subs`.

File: django_oidc_user/models.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    sub = models.CharField(max_length=255, unique=True)
```
Comment (ziima) [inline django_oidc_user/models.py]: `TextField` would be probably less problematic.

File: django_oidc_user/tests/settings.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+"""Settings for tests."""
+from __future__ import unicode_literals
+
+SECRET_KEY = 'Black hole!'
```
Comment (ziima) [inline django_oidc_user/tests/settings.py]: `Black hole sucks!`

**Comment by tpazderka** [comment]
Fixed.

**Pull request merged**